x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/render_as_select.tsx

```diff
@@ -5,19 +5,21 @@
  * 2.0.
  */
 
-import React from 'react';
-import { EuiFormRow, EuiComboBox, EuiComboBoxOptionOption } from '@elastic/eui';
+import React, { useState } from 'react';
+import { EuiFormRow, EuiButtonGroup } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { RENDER_AS } from '../../../../common/constants';
 
 const options = [
   {
+    id: 'clusters',
```

Comment:
nit: I'd just reuse the `RENDER_AS`-enumeration here, iso introducing new constants.